tests/lua_loader/tests/01-run.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2019 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+# Tell the lua interpreter running in riot to load some modules and print
+# the value of a variable inside that module.
+
+import os
+import sys
+
+MODULE_QUERIES = [
+    ("m1", "a", "1"),
```

Comment:
I just realized, if the expected value matches the module name, then the `expect_exact` will pass no matter the returned value, right?

Reply:
Dang! you are right! For some reason I thought expect_exact meant "expect exactly this". It seems I'm not be smart enough to use pexpect.

But wait, there's even more! Say I changed the 1 with a 9, then thanks to glorious pyterm the
"2019-05-31 ....!" at the beginning of a line would also be matched. It seems I'm not smart enough to use pyterm either.